setup.sh

```diff
@@ -21,7 +22,11 @@ if [ ! -e $env ]; then
 fi
 
 echo "=== Install Python packages into $env..."
-$env/bin/pip install sqlalchemy alembic pyyaml watchdog || exit 1
+$env/bin/pip install -r $codalabdir/requirements.txt || exit 1
+
+# Use sckoo@stanford.edu's patched argcomplete, until pull request has been accepted
+# https://github.com/kislyuk/argcomplete/pull/118
+$env/bin/pip install git+https://github.com/kashizui/argcomplete.git@allow-overridden-subparsersaction
```

Comment:
Ohh... that's interesting.
By the way could you explain or point me to documentation about how `codalab` is running code from my `codalab-cli` repo?